jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibPlugin.java

```diff
@@ -87,17 +87,36 @@ public void apply(Project project) {
     Task buildImageTask =
         project
             .getTasks()
-            .create(BUILD_IMAGE_TASK_NAME, BuildImageTask.class)
+            .create(
+                BUILD_IMAGE_TASK_NAME,
+                BuildImageTask.class,
+                task -> {
+                  task.setGroup("Jib Docker");
+                  task.setDescription("Builds a container image.");
```

Comment:
Maybe specify "Builds a container image to a registry"?

Reply:
[done](https://github.com/aivantsov/jib/commit/9c2ce6c918c8d8c041a2f8838a7fb05ed7aba982)